docs/development/codeguide.rst

```diff
@@ -556,12 +344,19 @@ Requirements Specific to Affiliated Packages
   conventions for an affiliated package are either simply ``packagename``
   or ``awastropy.packagename`` ("affiliated with Astropy").
 
+* If the affiliated package requires compatibility with Python 2, it should
+  pin to astropy<=2 until it is ready to drop Python 2 support.
```

Comment:
maybe pin to astropy<3? pip doesn't seem to pick up the latest LTS, but still with 2.0 with this syntax.